test/SourceKit/CodeComplete/complete_generic_property_wrapper_initialization.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %complete-test %s -tok=COMPLETE_GENERIC
+// RUN: %complete-test %s -tok=COMPLETE_IN_GENERIC_CONTEXT | %FileCheck --check-prefix=CHECK_COMPLETE_IN_GENERIC_CONTEXT %s
+
+@propertyWrapper public struct GenericWrapper<Value> {
+  public var wrappedValue: Value
+  public var projectedValue: Int
+
+  public init(wrappedValue: Value) {
+    self.wrappedValue = wrappedValue
+    self.projectedValue = 1
+  }
+}
+
+public struct GenericContext<T> {
+  @propertyWrapper public struct GenericWrapper<Value> {
+    public var wrappedValue: Value
+    public var projectedValue: Int
+
+    public init(wrappedValue: Value) {
+      self.wrappedValue = wrappedValue
+      self.projectedValue = 1
+    }
+  }
+}
+
+public struct MyStruct {
+  @GenericWrapper var someProperty = #^COMPLETE_GENERIC^#
+  @GenericContext<Int>.GenericWrapper var someProperty2 = #^COMPLETE_IN_GENERIC_CONTEXT^#
```

Comment:
When testing `COMPLETE_IN_GENERIC_CONTEXT`, other tokens are just erased from the source so,
```swift
   @GenericWrapper var someProperty = #^COMPLETE_GENERIC^#
   @GenericContext<Int>.GenericWrapper var someProperty2 = #^COMPLETE_IN_GENERIC_CONTEXT^#
```
is (probably) parsed as
```swift
@GenericWrapper var someProperty = @GenericContext<Int>.GenericWrapper /* missing ';' */ var someProperty2 = <TOKEN>
```
To prevent this, please wrap them separate types.

Reply:
👍 Moved them to two different types. Didn’t think about that.